### ISSUE TITLE
Sharing context - allow a null function? 

### ISSUE THREAD

**Issue by dwjohnston** [jxom/react-loads#20]
Hey Jake - 

I'm coming from using redux here, I'm wondering about how you go about sharing the context across multiple components. 

The idea is - when your application has loaded, its gone an fetched some data (say the user object), and then you might be using that user object everywhere - Displaying their avatar picture here, email address there, etc. 

I see that you can use the `context` option to share the data across. However, when using the `useLoads` hook, the typescript still requires you pass it an async function. 

Here's an example: 

```
//Our component where we're actually going to fetch the data. 
export const ThingyA: React.FunctionComponent<ThingyAProps> = (props) => {
    const getThingys = React.useCallback(_getThingys,[]); 
    const { response, error, load, isIdle,  isRejected, isPending, isResolved } = useLoads(getThingys, {
        defer: true, 
        context: "thingys", 
    });

    return <div >
      <h2> Thingy A</h2>
        {isIdle && <p> Idle</p>}
        {isPending &&  <p>...loading</p>}
        <button onClick = {load}> Get the things</button>
        {JSON.stringify(response)}
    </div>;
}

//Our other function where we're just displaying the data
export const ThingyB: React.FunctionComponent<ThingyBProps> = (props) => {
    const {
       
    } = props;
 
    //In this case I just pass it an empty async function, as I'm never going to use the load function  here. 
    const { response } = useLoads(async () => null, {
        defer: true, 
        context: "thingys"
    });
    return <div >
      <h2> Thingy B</h2>
      
      {JSON.stringify(response)}
      
    </div>;
}

```
Github with this example : https://github.com/dwjohnston/react-loads-play/tree/feature/context-sharing

Is there a tidier way to do this? Couldn't see a whole lot in the documentation re: context sharing.

**Comment by jxom** [comment]
Good point. If you don't want to pass in a loading function, maybe I could expose a hook to retrieve a record from context, such as:

```jsx
const thingys = useLoadsContext('thingys');
// thingys = { response: 'the-value' }
```

**Comment by jxom** [comment]
This should be published on `8.1.0` now. 

https://github.com/jxom/react-loads#cache--useloadscachecontext
